_build/builder/BuildIDEIntegration.py

```diff
@@ -47,26 +46,69 @@
 # with EventGhost. If not, see <http://www.gnu.org/licenses/>.
 
 """
-This file was automatically created by the BuildStaticImports.py script.
+This file was automatically created by the BuildIDEIntegration.py script.
 Don't try to edit this file yourself.
 
-This module is not directly used by EventGhost. It only exists to help
-pylint and other tools to read the sources properly, as EventGhost is using
-a lazy import pattern.
+This module is used to assist and IDE in locating the modules that are apart 
+of EventGhost. Because EventGhost uses a dynamic import system and there are 
+no "hard coded" imports of it's modules, normally an IDE would not be able to 
+track these modules. This file never gets used by the program, it is only here 
+to provide the "hard coded" imports an IDE needs to see.
+
+Follow the instructions below to create an EventGhost programming environment.
+Here is a list of a couple of the features in your IDE that will now work 
+properly.
+
+    Code Completion
+    Function/Method/Class parameters
+    Code Inspection
+    Object Definitions
+
+Instructions for PyCharm
+    Build EventGhost with the Build IDE Integration selected. 
+    File --> Settings --> Project:{YOUR PROJECT NAM} --> Project Structure
+    +Add Content Root
+    Navigate to one level lower then this file location.
```

Comment:
Not sure if i missunderstand of if it's wrong. My folder structure is this:
```
EventGhost-dev
    _build
        builder
            BuildIDEIntegration.py
        data
        ...
    eg
        Classes
        ...
```
From your instruction i would set `_build` as source folder, but ive set `EventGhost-dev` as sources folder. Can you clarify?

Reply:
what you are reading is the header that gets placed into StaticImports.py. which when built resides in the eg folder. so one folder then the eg folder is thee main directory for eg.. this way 

    import eg

will work

Reply:
Ah, yes, i ignored that it's written to another file.